.github/workflows/publish-documentation.yml

```diff
@@ -18,12 +18,10 @@ jobs:
       - name: Execution permission for Gradle wrapper
         run: chmod +x gradlew
       - name: Dokka
-        run: ./gradlew dokka
-      - name: Gather modules documentation
-        run: ./gradlew gatherDokkaDocumentation
+        run: ./gradlew dokkaHtmlMultiModule
```

Comment:
I've looked into it and we might actually prefer `dokkaHtmlCollector`, as it builds the documentation as if it was a single module. Makes the search and navigation much easier, but it still does not generate the top level `index.html` file for some reason.